src/components/SearchForm/SearchForm.jsx

```diff
@@ -10,22 +10,23 @@ const SearchForm = ({onSerch}) =>
       };
 
     const handleSubmit = (event)=>{
+        event.preventDefault(); 
         console.log(query);
         onSerch(query);
     }
 
     return (
         <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component